Document instance limiting (#2912)
Qiskit/documentation: docs/guides/allocation-limits.mdx

---
title: Set instance allocation limits
description: Set, view, and change how much time an instance is allowed to use.
platform: cloud
---

> # Set instance allocation limits

Reviewer: Allocation is mandatory but you can decide if the allocation will be limited or if you allow the instance to overconsume (so it being a limit is optional).

---
title: Set instance allocation limits
description: Set, view, and change how much time an instance is allowed to use.
platform: cloud
---
# Set instance allocation limits

> Premium accounts are alloted a certain amount of time.  Because each account can have many instances, setting allocation limits helps you control which instances are allowed to consume that time.

Reviewer: ```suggestion
An account may have allotted time on the premium plan. Because each account can have many premium plan instances, setting allocation limits helps you control which instances are allowed to consume that time.
```

> The time that you allocate is used by the fair-share algorithm and determines the priority in the queue, based on usage of all QPUs over a rolling time window of 28 days.

Reviewer: ```suggestion
The time that you allocate to premium plan instances is used by the fair-share algorithm and determines the priority in the queue, based on usage of all QPUs over a rolling time window of 28 days.
```

Optionally you can set this time allocation as a limit for the instance.

> Setting allocations and limits is optional.  If none are set for your instances, the [fair-share scheduler](/fair-share-queue) prioritizes workloads from instances with lower usage, and your account is still limited to its overall time allotment.

Reviewer: setting allocations is mandatory for premium plan instances
